tests/UI/pages/FO/myAccount/creditSlips.js

```diff
@@ -1,12 +1,21 @@
 require('module-alias/register');
 const FOBasePage = require('@pages/FO/FObasePage');
 
-class CreditSlips extends FOBasePage {
+/**
+ * Credit slip page, contains functions that can be used on the page
+ * @class
+ * @extends FOBasePage
+ */
+class CreditSlip extends FOBasePage {
```

Comment:
The file shouldn't be renamed ?

Reply:
if renamed, a lot of updates will have to be done. It will be on another PR.